sdk-extensions/autoconfigure/build.gradle.kts

```diff
@@ -14,7 +14,7 @@ testSets {
     create("testInitializeRegistersGlobal")
     create("testJaeger")
     create("testPrometheus")
-    create("testOtlpTls")
+    create("testOtlp")
```

Comment:
Previously there were OTLP specific tests in `testFullConfig` and `testOtlpTls`. I've renamed `testOtlpTls` => `testOtlp` and moved all OTLP specific tests there.

Reply:
Thanks seems to have been some bad-tasting copy pasta